### PR TITLE
Feature/sort by rating or name

### PR DESCRIPTION
**Pull request by MagnusTerak**
In this PR i've added the feature to sort the displaying challenges cards.

**Features made in the pull-request:** 
* Added sorting option to sort challenges cards by name or rating

**Changes made:**
* Modified the function executeSearch to handle the sorting

**Testing:** 
* Made sure all the filter functions still are working.
* Filter functions working combined with sorting options

**Images**

sortbig

sortcloseup

### CODE REVIEW

**Comment by MagnusTerak** [comment]
[Sortera rum på rating eller namn](https://trello.com/c/K37mreSe/10-sortera-rum-p%C3%A5-rating-eller-namn)

**Review by Deansie — COMMENTED** [review]
The sorting menu looks really good, well done! However, I wonder if we should perhaps stick with the language of our HTML document, so instead of A-Ö it should say A-Z.

After selecting a sorting option, when you click on any other filtering options, the sorting from your menu is reset. It would be nice if the sorting from your menu persisted even after other options are chosen.

**Review by El-Duckster — CHANGES_REQUESTED** [review]

File: src/js/challengeClass.js [hunk 1]
```diff
@@ -109,7 +108,7 @@ class Challenge {
     
     const challengeDescription = document.createElement("p");
     challengeDescription.className = "challenge__description"
-    challengeDescription.textContent = truncateText(this.description, 50);
+    challengeDescription.textContent = this.description;
```
Comment (El-Duckster) [inline src/js/challengeClass.js:111]: here too this will overrwritte the new changes.

File: src/js/challengeClass.js [hunk 2]
```diff
@@ -1,6 +1,5 @@
 import'../styles/layouts/rooms.scss';
 import {openBookingModal} from './modal1.js'
-import {truncateText} from './utilities/truncateText.js';
```
Comment (El-Duckster) [inline src/js/challengeClass.js:3]: here :)

**Review by El-Duckster — CHANGES_REQUESTED** [review]

**Comment by MagnusTerak** [comment]
This PR & Branch has gotten backtracked due to merges, we've decided to close this and delete the branch (New branch & PR with correct version has been created)

**Pull request closed without merging**